### ISSUE TITLE
Buttonbox Color field

### ISSUE THREAD

**Issue by DenyEs** [supercool/buttonbox#45]
Buttonbox color field  causes javascript related issues when I first try to add a block to an entry. It makes it even worse when it's inside of a supertable as it's stopping the rest of javascript work, all the fields won't stack but they spill out of the container. When I refresh a page, it's all good but when I add another field that contains buttonbox color field it's the same issue again unless I refresh or re-save an entry. Issue can be seen on the attached screenshot

**Steps to reproduce**
1. Create a buttonbox color field
2. Assign a field to an entry type
3. Open the console
4. There should be an error related to $container

**Additional info**
- Plugin version: 4.0.3
- Craft version: 4.0.5.2
- Multi-site: N
<img width="1928" alt="buttonbox-color-field-issue" src="https://user-images.githubusercontent.com/18517642/176131919-8c2800e9-5b18-4b2a-84eb-5d58accf2940.png">

**Comment by engram-design** [comment]
I'm not seeing this error when using any Button box field, sorry. The only instance where this is happening is within a Super Table field.

Fixed for the next release. To get this early, change your `verbb/buttonbox` requirement in `composer.json` to:

Craft 3
```
"require": {
  "verbb/buttonbox": "dev-craft-3 as 3.0.2",
  "...": "..."
}
```

Craft 4
```
"require": {
  "verbb/buttonbox": "dev-craft-4 as 4.0.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by DenyEs** [comment]
Thank you Josh for fixing it so quickly! That fixed it for me in both supertable fields and outside of them as well. Appreciate it

**Comment by engram-design** [comment]
Fixed in [3.1.0](https://github.com/verbb/buttonbox/releases/tag/3.1.0) and [4.1.0](https://github.com/verbb/buttonbox/releases/tag/4.1.0).
